config/v201/config.json

```diff
@@ -209,13 +209,13 @@
             "SupportedCiphers12": {
                 "variable_name": "SupportedCiphers12",
                 "attributes": {
-                    "Actual": "ECDHE-ECDSA-AES128-GCM-SHA256,ECDHE-ECDSA-AES256,GCM-SHA384,AES128-GCM-SHA256,AES256-GCM-SHA384,TLS_AES_256_GCM_SHA384,TLS_AES_128_GCM_SHA256"
+                    "Actual": "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:AES128-GCM-SHA256:AES256-GCM-SHA384"
                 }
             },
             "SupportedCiphers13": {
                 "variable_name": "SupportedCiphers13",
                 "attributes": {
-                    "Actual": "TLS_AES_256_GCM_SHA384,TLS_AES_128_GCM_SHA256"
+                    "Actual": "TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256"
                 }
```

Comment:
I suggest we remove these configs here. There should be only one single source of truth, also for config-defaults.

If you really need to change this, then you should definitely know and understand why you're doing this anyway. Then you'll also figure out how to set this config.

Reply:
Sure but that seems out of scope for this fix, so I would prefer to leave this change in this PR.